### DISCUSSION TITLE
Build fails without details

### DISCUSSION THREAD

**Discussion by deleteman** [vercel/community#2950] [Help] [answered]
### Summary

I'm currently trying to deploy my application, and while it's been working for months perfectly, now the deploy fails and the only error I get is: Command "npm run build" exited with 1

There are no build logs, or anything I can look at to understand what's wrong with the build process.

### Steps to Reproduce

1. Push to github
2. Build fails automatically after a few seconds

**Comment by amyegan** [comment, marked as answer]
Hi @deleteman. If you share the project id or a link to the failed deployment, then I can took for more info on what went wrong.

**Reply by deleteman** [reply]
Hey @amyegan , here is one of those failed deployments: https://vercel.com/deleteman/virtualsnap-fa5w/AS67ySUAVFjRNe1scRfbjcYHYct5

**Reply by amyegan** [reply]
Thanks for sharing that. I looked up the build log and see there are some warnings about React Hook `useEffect` missing dependencies. There are also a few `Module not found` errors about `@sass-ui/react` and `@saas-ui/charts`.

These errors commonly happen if a dependency is missing from `package.json`, but there are other potential causes as well. You can find more info on possible causes and solutions here: https://nextjs.org/docs/messages/module-not-found
